### ISSUE TITLE
Feature request: mrtransform : transformation/downsampling using modus without interpolation for atlas / parcellation masks.

### ISSUE THREAD

**Issue by JBaijot** [MRtrix3/mrtrix3#2018]
As written on the website under the topic: "How do I use atlas / parcellation “X”?"
The recommendation would be to transform the  atlas / parcellation to the subject space.
As suggested in the given code a transformation matrix can be obtained and applied to the atlas with the "mrtransform" function. Issues rise with re-sampling to the subject space and the interpolation method as parcel number 1 can be next to parcel number 100 with all interpolation methods this will result in a voxel at the border with a different value, let say 50, which we do not want as 50 is part of another parcel. Would i be possible to have a interpolation method that assigns 0,1 or 100 to this pixel.
In other words I would need an interpolation method delivering the modus of the voxels being down-sampled.

**Comment by Lestropie** [comment]
Looks like the description includes the solution, but the code example doesn't:

> When re-sampling a parcellation image to a different image grid, nearest-neighbour interpolation must be used; otherwise the underlying integer values that correspond to parcel identification indices will be lost.

`mrtransform` should be run with `-interp nearest` in this context.

**Comment by JBaijot** [comment]
Thanks that is also what I am currently using. My confusing started started when I looked at the resulting images, but I know figured out that MRview also applies an interpolation between the pixels it renders.

**Comment by Lestropie** [comment]
Yes, that catches people out. Following imminent release, with #1568 the fact that the reported value at the focus point is based on interpolation if the display also uses interpolation will hopefully be more clear.
